src/components/country-state-item.js

```diff
@@ -85,42 +61,29 @@ const CountryStateItem = (props) => {
                     xs={12}
                     style={{ display: 'flex', width: '100%', padding: 8 }}
                 >
-                    {status.status === IgmStatus.ABSENT ? (
-                        <HourglassEmptyIcon
-                            className={`${classes.stateIcon} ${classes.waiting}`}
-                        />
-                    ) : status.status === IgmStatus.AVAILABLE ? (
-                        <LoopIcon
-                            className={`${classes.stateIcon} ${classes.loading}`}
-                        />
-                    ) : status.status === IgmStatus.INVALID ? (
-                        <WarningIcon
-                            className={`${classes.stateIcon} ${classes.error}`}
-                        />
-                    ) : status.status === IgmStatus.VALID ? (
-                        <MergeIcon
-                            className={`${classes.stateIcon} ${classes.success}`}
-                        />
-                    ) : status.status === IgmStatus.MERGED ? (
-                        <DoneIcon
-                            color="secondary"
-                            className={`${classes.stateIcon} ${classes.success}`}
-                        />
-                    ) : (
-                        ''
-                    )}
-                    <Box className={classes.textColumn}>
-                        <Typography variant="body1">
+                    <LensIcon
+                        className={classes.stateIcon}
+                        style={{ color: color }}
+                    />
+                    <Box className={classes.tsosColumn}>
+                        <Typography variant="body1">{props.tso}</Typography>
+                        <Typography variant="caption">
                             {detail.countryName}
                         </Typography>
                     </Box>
-                    <img
-                        className={classes.flagIcon}
-                        alt="flag for country"
-                        height="23"
-                        width="32"
-                        src={detail.flagSrc}
-                    />
+                    {replacingDate && (
+                        <Box className={classes.textReplace}>
+                            <Typography
+                                variant="body1"
+                                className={classes.smallText}
+                            >
+                                {intl.formatMessage({ id: 'ReplacedWith' })}
```

Comment:
In my opinion, the replacing business process should also be included in the 'replaced with' part